v3/plugins/vuejs/vetur/0.26.1/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: vuejs
+name: vetur
+version: 0.26.1
+type: VS Code extension
+displayName: Vetur
+title: Rich Vue.js support
+description: Vue tooling for VS Code
+icon: https://upload.wikimedia.org/wikipedia/commons/9/95/Vue.js_Logo_2.svg
```

Comment:
Could you apply this image instead of getting it from wikimedia.org
https://raw.githubusercontent.com/vuejs/vetur/v0.26.1/asset/vue.png
?